opium_kernel/core/rock.ml

```diff
@@ -27,13 +27,23 @@ module Pp = struct
         ])
```

Comment:
General comment about the diff for this file. I'd lean towards using the existing pretty printers (pp_hum) functions instead of something that just converts things to a string. The formatters are more flexible and still allow converting content into a string via `Format.asprintf "%a" M.pp_hum`